src/agent/debuglet.ts

```diff
@@ -291,6 +291,19 @@ export class Debuglet extends EventEmitter {
     const that = this;
     const stat = promisify(fs.stat);
 
+    const workingDir = that.config.workingDirectory;
+    // Don't continue if the working directory is a root directory
+    if (path.join(workingDir, '..') === workingDir) {
+      that.logger.error(
+          `Refusing to start with \`workingDirectory\` set to a root directory:  '${
+              workingDir}'`);
+      that.emit(
+          'initError',
+          new Error(
+              `Cannot start the agent when the working directory is a root directory`));
```

Comment:
1. This doesn't tell the user why we don't want to start if we're in the root directory.
1. Please provide a way (config) for users to silence this check and force a start even in the root directory.
1. The check for package.json should happen before this check. That is a more severe problem.

Reply:
Fixed 1 and 3.

Reply:
Fixed 2